eclair-core/src/main/scala/fr/acinq/eclair/blockchain/bitcoind/ZmqWatcher.scala

```diff
@@ -159,14 +160,14 @@ object ZmqWatcher {
   private case object Ignore extends AddWatchResult
   // @formatter:on
 
-  def apply(chainHash: ByteVector32, blockCount: AtomicLong, client: ExtendedBitcoinClient): Behavior[Command] =
+  def apply(chainHash: ByteVector32, blockCount: AtomicLong, client: ExtendedBitcoinClient, socksProxy_opt: Option[Socks5ProxyParams], watchdogBlacklist: Seq[String]): Behavior[Command] =
```

Comment:
I'd rather remove `chainHash`, `socksProxy_opt` and `watchdogBlacklist` and instead provide the whole `nodeParams` object, as we do in other actors. Same for the children watchdog actors.